export/client.go

```diff
@@ -0,0 +1,38 @@
+package export
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	terra "github.com/terra-money/core/v2/app"
+)
+
+func ShouldCacheRoute(route string) bool {
```

Comment:
Why did you name this function like this? I guess `ShouldHandleRoute` is more proper..?

Reply:
Currently all endpoints expect `/health` is cached and I wanted to exclude exports from the cache. But thinking through it, there is no reason not to cache `circulating_supply`. Let me remove this.